packages/kolibri-common/components/SearchFiltersPanel/index.vue

```diff
@@ -214,8 +214,6 @@
         searchableLabels,
         activeSearchTerms,
         windowIsLarge,
-        // This color is not in KDS but was specifically requested in the design
```

Comment:
But then also never used here! cc @nucleogenesis

Reply:
Missed this during clean-up - thankful for the new maximally strict checker :)